packages/apps/graph/src/services/node-service.ts

```diff
@@ -30,84 +32,93 @@ export type ChainFungibleAccountDetails = {
   };
 };
 
-function getClient(chainId: ChainId): IClient {
-  return createClient(
-    `http://${dotenv.NETWORK_HOST}/chainweb/0.0/${dotenv.NETWORK_ID}/chain/${chainId}/pact`,
-  );
-}
-
 export async function getAccountDetails(
   fungibleName: string,
   accountName: string,
   chainId: string,
 ): Promise<ChainFungibleAccountDetails | null> {
-  const commandResult = await getClient(chainId as ChainId).dirtyRead(
-    Pact.builder
-      .execution(
-        Pact.modules[fungibleName as 'fungible-v2'].details(accountName),
-      )
-      .setMeta({
-        chainId: chainId as ChainId,
-      })
-      .setNetworkId(dotenv.NETWORK_ID)
-      .createTransaction(),
-  );
+  let result;
+  try {
+    result = (await details(
+      accountName,
+      dotenv.NETWORK_ID,
+      chainId as ChainId,
+      dotenv.NETWORK_HOST,
+      fungibleName,
+    )) as any;
```

Comment:
Why did you have to cast it to any?

Reply:
Because Typescript doesn't like the return type: 
![image](https://github.com/kadena-community/kadena.js/assets/6686385/ee6ce4f8-98c2-4349-a9c9-fb4d07a5066b)

Reply:
I see, 
I have addressed this issue in this PR https://github.com/kadena-community/kadena.js/pull/1321. 
I'll let you. know when I merge it